src/Illuminate/Database/DetectsConcurrencyErrors.php

```diff
@@ -16,6 +36,13 @@ trait DetectsConcurrencyErrors
      */
     protected function causedByConcurrencyError(Throwable $e)
     {
+        if (
+            is_callable($this->concurrencyErrorCheck)
+            && call_user_func($this->concurrencyErrorCheck, $e)
```

Comment:
just call it with native syntax

Reply:
Done. I just haven't seen that syntax used as much as this when I've looked at the framework code :)